src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
```

Comment:
I checked the behaviour, this call ignores the case, so we don't need to do a ToLower or anything like that. 👍